gst-plugin-pravega/src/timestampcvt/imp.rs

```diff
@@ -44,33 +47,44 @@ pub enum InputTimestampMode {
         nick = "ntp"
```

Comment:
Will rtspsrc element whose `ntp-time-source` property set to `running-time` and `ntp` using the same time format(NTP time)  when writing PTS into buffer?

Reply:
PTS always stores nanoseconds. With `running-time`, the first PTS of the pipeline is 0. With `ntp`, the PTS is the number of nanoseconds since the NTP epoch 1900-01-01 UTC.

Reply:
In this case, the description above(line 44-45) is not accurate.  If ntp-time-source=running-time, the input buffer timestamps are not nanoseconds since the NTP epoch.